util/parsenum.h

```diff
@@ -143,6 +143,9 @@ parsenum_float(const char * s, double min, double max, int trailing)
 	char * eptr;
 	double val;
 
+	/* Sanity check. */
```

Comment:
This commit is fine, but the message talks about `optarg` which seems a bit of a red herring?

I mean, ok, `s` is `optarg` if you're using this to parse command-line arguments, but you shouldn't need to understand about getopt in order to understand a commit to parsenum.

Reply:
Ok, I removed the detailed commit message.

... even though I was so proud of my wordplay about "most often"!

Reply:
Umm... the details were fine.  My objection is that the top line says "check if optarg is NULL" and there's no optarg here.